### PR TITLE
Feature/change structure

### PR DESCRIPTION
**Pull request by Sergyo97**

### CODE REVIEW

File: .factorypath [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<factorypath>
```
Comment (Alejoguzm07) [inline .factorypath]: Is this necessary? maybe you should add something to .gitignore file

File: src/main/java/eci/ieti/leafOut/backEnd/controller/ParkController.java [hunk 2]
```diff
@@ -19,21 +20,16 @@
...
 
     @GetMapping
-    public ResponseEntity<?> getAllPlans(){
-        try{
-            List<Park> parks = parkServices.getAllParks();
-            return new ResponseEntity<>(parks, HttpStatus.ACCEPTED);
-        }catch (LeafOutPersistenceException ex){
-            return new ResponseEntity<>(ex.getMessage(),HttpStatus.INTERNAL_SERVER_ERROR);
-        }
+    public ResponseEntity<?> getAllPlans() {
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/controller/ParkController.java:23]: Javadoc

File: src/main/java/eci/ieti/leafOut/backEnd/controller/PaymentsController.java [hunk 3]
```diff
@@ -21,7 +21,7 @@
      * Injected payments service instance
      */
     @Autowired
-    private PaymentsService paymentServices;
+    PaymentsService paymentServices;
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/controller/PaymentsController.java:24]: Access modifiers

File: src/main/java/eci/ieti/leafOut/backEnd/model/Park.java [hunk 4]
```diff
@@ -12,9 +12,7 @@
     private String addressPark;
     private List<Plan> plans;
 
-    public Park(){
-
-    }
+    public Park(){ }
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/model/Park.java:15]: Consider using lombok

File: src/main/java/eci/ieti/leafOut/backEnd/model/User.java [hunk 5]
```diff
@@ -2,42 +2,4 @@
 
 public class User {
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/model/User.java]: Don't delete teammate's code

File: src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java [hunk 6]
```diff
@@ -1,17 +1,35 @@
...
+import eci.ieti.leafout.backend.repository.PlanRepository;
+
+@Service
+/**
+ * PlanService
+ */
+public class PlanService {
+
+    @Autowired
+    PlanRepository planRepository;
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java:20]: Modifiers

File: src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java [hunk 7]
```diff
@@ -1,17 +1,35 @@
...
+import org.springframework.stereotype.Service;
+
+import eci.ieti.leafout.backend.model.Plan;
+import eci.ieti.leafout.backend.repository.PlanRepository;
+
+@Service
+/**
+ * PlanService
+ */
+public class PlanService {
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java:17]: Should implement an interface

**Review by luis08a — APPROVED** [review]

File: src/main/java/eci/ieti/leafOut/backEnd/service/impl/ParkServiceImpl.java [hunk 8]
```diff
@@ -1,39 +0,0 @@
-package eci.ieti.leafOut.backEnd.service.impl;
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/service/impl/ParkServiceImpl.java:1]: Don't delete

File: src/main/java/eci/ieti/leafOut/backEnd/service/impl/PaymentsServiceImpl.java [hunk 9]
```diff
@@ -1,48 +0,0 @@
-package eci.ieti.leafout.backend.service.impl;
```
Comment (Alejoguzm07) [inline src/main/java/eci/ieti/leafOut/backEnd/service/impl/PaymentsServiceImpl.java:1]: Don`t delete

**Review by Alejoguzm07 — CHANGES_REQUESTED** [review]

File: src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java (hunk 6 above)
Comment (Sergyo97) [reply src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java:20]: To solve by @QSARJP

File: src/main/java/eci/ieti/leafOut/backEnd/model/Park.java (hunk 4 above)
Comment (Sergyo97) [reply src/main/java/eci/ieti/leafOut/backEnd/model/Park.java:15]: To solve by @QSARJP

File: src/main/java/eci/ieti/leafOut/backEnd/controller/ParkController.java (hunk 2 above)
Comment (Sergyo97) [reply src/main/java/eci/ieti/leafOut/backEnd/controller/ParkController.java:23]: To solve by @QSARJP

File: src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java (hunk 6 above)
Comment (Alejoguzm07) [reply src/main/java/eci/ieti/leafOut/backEnd/service/PlanService.java:20]: ok

**Pull request closed without merging**